### PR TITLE
[SD-758] Removes input field shadow for ios

### PR DESCRIPTION
**Pull request by PSzyszkaSpark**
Fixes https://spark-solutions.atlassian.net/browse/SD-758

### CODE REVIEW

File: frontend/app/assets/stylesheets/spree/frontend/views/spree/orders/edit.scss [hunk 1]
```diff
@@ -354,6 +359,11 @@
         }
       }
     }
+    input[type=text] {
```
Comment (damianlegawiec) [inline frontend/app/assets/stylesheets/spree/frontend/views/spree/orders/edit.scss]: @PSzyszkaSpark it's not only text but also, email, password, number

**Review by damianlegawiec — CHANGES_REQUESTED** [review]

File: frontend/app/assets/stylesheets/spree/frontend/views/spree/orders/edit.scss (hunk 1 above)
Comment (PSzyszkaSpark) [reply frontend/app/assets/stylesheets/spree/frontend/views/spree/orders/edit.scss]: @damianlegawiec I've added email, password, number types to separate locations, where they needed to be placed. In cart there are only inputs with type string.

**Review by pmilewski — CHANGES_REQUESTED** [review]
I found a few places in the app when this issue occurs (eq. edit your account page, forgot password). I think we should apply this fix **globally** for all text, email, password and number inputs.

**Review by pmilewski — APPROVED** [review]

**Comment by pmilewski** [comment]
Please squash commits.

**Pull request merged**